nat-lab/tests/test_dns.py

```diff
@@ -190,36 +190,58 @@ async def test_dns_port(alpha_ip_stack: IPStack) -> None:
         connection_alpha, _ = [conn.connection for conn in env.connections]
 
         # These call should timeout without returning anything
-        with pytest.raises(asyncio.TimeoutError):
-            await testing.wait_normal(
+        with pytest.raises(ProcessExecError):
+            await testing.wait_long(
                 connection_alpha.create_process(
-                    ["dig", "@" + dns_server_address_alpha, "-p", "53", "google.com"]
+                    [
+                        "dig",
+                        "+timeout=1",
+                        "@" + dns_server_address_alpha,
+                        "-p",
+                        "53",
+                        "google.com",
+                    ]
```

Comment:
As a possible improvement I think we could use `dig` with one of our own services inside of the docker, what do you think about that?

Reply:
It's already doing that. `dig` will communicate with the node's internal DNS address, which will delegate to, in this case, the DNS docker container that Tomasz created for anything that is not looking up the IP of a meshnet peer

Reply:
As an improvement I agree however for this PR might be already out of scope

Reply:
Ooops! Google.com is just a placeholder, `dns_server_address_alpha` is already there :facepalm: